pkg/daemon/controller/node.go

```diff
@@ -79,25 +81,26 @@ func (nic *NodeIPCache) UpdateNodeIPs(nodeList []*v1.Node, localNodeName string)
 		}
 	}
 
+	for _, remoteNode := range remoteNodeList {
```

Comment:
ditto, node.Annotations[constants.AnnotationNodeLocalVxlanIPList] should also be updated to RemoteVtep CRs.